### PR TITLE
[Docs] batch 03 - trig functions

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

Add examples to the sql-functions.md page

Updates  the SIN, COS, TAN, COT, ASIN, ACOS, ATAN, ATAN2, DEGREES, and RADIAN functions

This PR has:

- [X] been self-reviewed.

### CODE REVIEW

File: docs/querying/sql-functions.md [hunk 1]
```diff
@@ -280,27 +295,72 @@ Joins all elements of `arr` by the delimiter specified by `str`.
...
+| `arc_sine` |  
+| -- | 
+| `1.5707963267948966` |
+</details>
+
+[Learn more](sql-scalar.md#numeric-functions)
+
+## ATAN
 
 Calculates the arc tangent of a numeric expression.
```
Comment (techdocsmith) [inline docs/querying/sql-functions.md:321]: Why are some functions explicitly called out as `trigonometric` (line 577) and others not?

**Review by techdocsmith — CHANGES_REQUESTED** [review]

File: docs/querying/sql-functions.md (hunk 1 above)
Comment (edgar2020) [reply docs/querying/sql-functions.md:321]: Agreed to keep as is to match definitions on [scalar functions page](https://druid.apache.org/docs/latest/querying/sql-scalar/#numeric-functions)

File: docs/querying/sql-functions.md [hunk 2]
```diff
@@ -514,20 +574,50 @@ Finds whether a string is in a given expression, case-sensitive.
...
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `COS(expr)`
+* **Function type:** Scalar, numeric
 
-Calculates the trigonometric cosine of an angle expressed in radians.
+<details><summary>Example</summary>
 
-## COT
+The following example calculates the cosine of `PI` over three radians.
```
[the remark this answers is not in the document]
Comment (edgar2020) [reply docs/querying/sql-functions.md]: Added "the angle" or equivalent to the functions SIN, COS, TAN, COT, DEGREES, and RADIANS. 

Also applied numeric notation to all functions. Made all values in code notation such as "The following example calculates the arc sine of `1`."

File: docs/querying/sql-functions.md [hunk 3]
```diff
@@ -70,11 +70,26 @@ Returns the following:
...
+Calculates the arc cosine of a numeric expression.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `ACOS(expr)`
+* **Function type:** Scalar, numeric
 
-Calculates the arc cosine of a numeric expression.
+<details><summary>Example</summary>
+
+The following example calculates the arc cosine of zero.
```
[the remark this answers is not in the document]
Comment (edgar2020) [reply docs/querying/sql-functions.md]: I included variant spelling to all "arc ______" functions. Added it to the definition and not the example explanation. Matches you suggestion for arcsine

File: docs/querying/sql-functions.md [hunk 4]
```diff
@@ -280,27 +295,72 @@ Joins all elements of `arr` by the delimiter specified by `str`.
 
 ## ASIN
 
-`ASIN(<NUMERIC>)`
+Calculates the arc sine of a numeric expression.
```
[the remark this answers is not in the document]
Comment (edgar2020) [reply docs/querying/sql-functions.md]: made all spelling alternatives match this example

**Review by techdocsmith — APPROVED** [review]
LGTM

**Pull request merged**